app/templates/custom-elements/button-dropdown.html

```diff
@@ -43,11 +43,9 @@
   </ul>
 </template>
 
-<script>
+<script type="module">
   (function () {
```

Comment:
I forgot to mention one thing: we don’t need to wrap the code into an IIFE (immediately invoked function expression) anymore, because it’s a module now. So we don’t pollute the global namespace e.g. with the `template` variable, since everything is enclosed in the module.

We could remove the IIFEs everywhere now, but that changes all the indentation. This not just results in a large change set, but we also “loose” (veil) the git history for the individual lines, when doing `git blame`. On the other hand, we could also leave it be: the IIFE doesn’t do any harm, since the webcomponent’s JS is not imported from anywhere else.

What do you think? I’d personally tend to leaving it, the legacy burden is rather insignificant.

Reply:
(And if at all, I’d also do this in separate PRs, not here.)